src/collective/instancebehavior/tests/test_instancebehavior.py

```diff
@@ -1,4 +1,5 @@
-import unittest2 as unittest
+# -*- coding: utf-8 -*-
```

Comment:
Not that it concerns me, but coding headers make only sense in files containing non ASCII chars. Seen this quite often recently that people add this headers without any need...